### ISSUE TITLE
Change return type of `import_block` to a NamedTuple

### ISSUE THREAD

**Issue by cburgdorf** [ethereum/py-evm#1881]
### What is wrong?

Currently the definiton of `import_block` looks like this:

```
    @abstractmethod
    def import_block(self,
                     block: BlockAPI,
                     perform_validation: bool=True,
                     ) -> Tuple[BlockAPI, Tuple[BlockAPI, ...], Tuple[BlockAPI, ...]]:
        """
        Imports complete block and returns a 3-tuple

        - the imported block
        - a tuple of blocks which are now part of the canonical chain.
        - a tuple of blocks which were canonical and now are no longer canonical.
        """
        ...
```
I believe the return type made sense under Python 3.5 but could be improved now that Py-EVM assumes Py 3.6+.

### How can it be fixed

Change the return type to something like:

```python

class BlockImportResult(NamedTuple):
    imported_block: BlockAPI
    new_canonical_blocks: Tuple[BlockAPI, ...]
    non_canonical_blocks: Tuple[BlockAPI, ...]
```

**Comment by Peppece** [comment]
is this fixed or not?

**Comment by cburgdorf** [comment]
Oh yeah, that was indeed fixed by #1910. Thanks for reporting :pray:
